pkg/cloudproviders/awsprovider/templates/controlPlane.yaml

```diff
@@ -0,0 +1,470 @@
+    Properties:
+      GroupName: !Join ["-", [!Ref 'AWS::StackName', "cp", "sg"]]
+      GroupDescription: Satellite Connectivity for Control Plane
+      VpcId: !Ref CPVPC
+      SecurityGroupIngress:
+        - CidrIp: 0.0.0.0/0
+          IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+        - CidrIp: 0.0.0.0/0
+          IpProtocol: tcp
+          FromPort: 443
+          ToPort: 443
+        - CidrIp: 0.0.0.0/0
+          IpProtocol: tcp
+          FromPort: 30000
+          ToPort: 32767
+        - CidrIp: 0.0.0.0/0
+          IpProtocol: udp
+          FromPort: 30000
+          ToPort: 32767
+      Tags:
+        - Key: "Name"
+          Value: !Join ["-", [!Ref 'AWS::StackName', "cp", "sg"]]
+
+  SelfSecurityGroupIngress:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId: !Ref SecurityGroup
+      SourceSecurityGroupId: !Ref SecurityGroup
+      IpProtocol: -1
+
+  EC2Role:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: ec2.amazonaws.com
+            Action: sts:AssumeRole
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess
+
+  InstanceProfile:
+    Type: "AWS::IAM::InstanceProfile"
+    Properties:
+      Roles:
+        - !Ref EC2Role
+
+  ControlPlaneLaunchTemplate:
+    Type: AWS::EC2::LaunchTemplate
+    Metadata:
+      AWS::CloudFormation::Authentication:
+        S3AccessCreds:
+          type: "S3"
+          roleName: !Ref EC2Role
+      AWS::CloudFormation::Init:
+        configSets:
+          init:
+            - repoKey
+        repoKey:
+          files:
+            "/root/attachHost.sh":
+              mode: '000100'
+              owner: root
+              group: root
+              source: !Ref IgnitionScriptS3URL
+              authentication: "S3AccessCreds"
+    Properties:
+      LaunchTemplateName: !Join ["-", [!Ref 'AWS::StackName', "ControlPlaneLaunchTemplate"]]
+      LaunchTemplateData:
+        BlockDeviceMappings:
+          - Ebs:
+              VolumeSize: 25
+              VolumeType: gp3
+              Iops: 3000
+              DeleteOnTermination: true
+              Encrypted: false
+            DeviceName: /dev/sda1
+          - Ebs:
+              VolumeSize: 100
+              VolumeType: gp3
+              Iops: 3000
+              DeleteOnTermination: true
+              Encrypted: false
+            DeviceName: /dev/sdb
+        ImageId: !Ref ImageID
+        InstanceType: !Ref InstanceType
+        IamInstanceProfile:
+          Name: !Ref InstanceProfile
+        SecurityGroupIds:
+          - !Ref SecurityGroup
+        UserData:
+          Fn::Base64:
+            Fn::Sub:
+              |
+              #!/bin/bash -xe
+              # install the packages required for cfn
+              sudo yum -y update
+              sudo yum -y install python3 python3-pip
```

Comment:
- pls cross check how this was being done in `attachHost` script

Reply:
attach host script install python3.9, I will update our script to have python3.9 as well